app/controllers/categories/index.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+const { Controller, inject: { service } } = Ember;
+
+export default Controller.extend({
+  router: service(),
+  actions: {
+    toNew() {
```

Comment:
I wanted to use paper-button and I failed to see a neater way to make it a 'link' than triggering an action on click